setup.py

```diff
@@ -8,6 +9,15 @@
     README = f.read()
 
 
+extra_kwargs = dict(use_scm_version=True)
+
+if subprocess.call(
+        "git rev-parse", shell=True,
+        stderr=subprocess.DEVNULL) != 0:
+    print("Disabling setuptools-scm")
+    extra_kwargs['use_scm_version'] = False
+
+
```

Comment:
Sorry, late review!

This and the related changes to disable setuptools-scm will actually cause problems, because the manifest will no longer be generated correctly. It should be better to keep setuptools-scm always enabled, and rather install git inside the dev container, so that everything works together.

I'll create a follow-up PR with these changes.